src/rtc/h5_prep.py

```diff
@@ -845,7 +850,7 @@ def get_metadata_dict(product_id: str,
              '_dem_egm_model',
              ALL_PRODUCTS,
              'Earth Gravitational Model EGM08',
```

Comment:
I think`Earth Gravitational Model` duplicates with `EGM`. Consider taking out one of these.

Reply:
Thank you, @seongsujeong